docs-preview/media/alert-styles.js

```diff
@@ -1,19 +1,19 @@
-var blockquoteTags = document.getElementsByTagName('blockquote');
+var blockquoteTags = document.getElementsByTagName("blockquote");
 
```

Comment:
Just an idea... but maybe you could make an array filled with strings of the different kinds of alerts, and then map a function onto the array to check for each kind, and assign its corresponding attribute?

Let me try working on an example...

Reply:
@jamarw  - what about something like this?

`var blockquoteTags = document.getElementsByTagName("blockquote");
var alerts = ["NOTE", "TIP", "IMPORTANT", "CAUTION", "WARNING"];

alerts.map(aa => {
  for (var i = 0; i < blockquoteTags.length; i++) {
    let tag = blockquoteTags[i];
    if (tag.innerHTML.toUpperCase().indexOf(`[!${aa}]`) > -1) {
      tag.setAttribute("class", aa);
    }
  }
});`

Reply:
@adunndevster Great idea!  I looked at a few options similar to this and didn't come up with anything as clean.  Thx